todoman/ui.py

```diff
@@ -224,7 +222,14 @@ def compact_multiple(self, todos):
             percent = todo.percent_complete or ''
             if percent:
                 percent = " ({}%)".format(percent)
-            urgent = " " if todo.priority in [None, 0] else "!"
+            if todo.priority == 5:
+                priority = "!!"
+            elif todo.priority <= 4 and todo.priority >= 1:
+                priority = "!!!"
+            elif todo.priority <= 9 and todo.priority >= 6:
+                priority = "!"
+            elif todo.priority == 0:
+                priority = " "
```

Comment:
(Also, if it's an empty string we don't add a column if no todos have priority).